plugins/remoteaccess/RemoteAccessWidget.cpp

```diff
@@ -372,3 +373,19 @@ void RemoteAccessWidget::takeScreenshot()
 {
 	Screenshot().take( m_computerControlInterface );
 }
+
+
+
+void RemoteAccessWidget::updateRemoteAccessTitle()
+{
+	if ( m_computerControlInterface->userFullName().isEmpty() )
+	{
+		setWindowTitle( tr( "%1 - %2 Remote Access" ).arg( m_computerControlInterface->computer().name(),
+														   VeyonCore::applicationName() ) );
+	} else
+	{
+		setWindowTitle( tr( "%1 - %2 - %3 - Remote Access" ).arg( m_computerControlInterface->userFullName(),
```

Comment:
There's an extra dash before "Remote Access" which should not be there. Except for this, the patch looks fine, thanks!